srcpkgs/EternalTerminal/template

```diff
@@ -1,29 +1,56 @@
 # Template file for 'EternalTerminal'
 pkgname=EternalTerminal
-version=6.0.13
-revision=7
+version=6.2.1
+revision=1
+_sanitizers_gitrev="99e159ec9bc8dd362b08d18436bd40ff0648417b"
+_threadpool_gitrev="9a42ec1329f259a5f4881a291db1dcb8f2ad9040"
```

Comment:
could you add a comment where you got these hashes from?

Reply:
Compliance.

Are these comments understandable?